contracts/governance/Staking/StakingStorage.sol

```diff
@@ -123,7 +123,7 @@ contract StakingStorage is Ownable {
     mapping(address => bool) public vestingWhitelist;
 
     /// @dev user => flag whether user has admin role.
-    /// @dev multisig should be an admin, admin can invoke only governanceWithdrawVesting function,
+    /// @dev multisig should be an admin, admin can invoke only governanceDirectWithdrawVesting function,
     /// 	this function works only with Team Vesting contracts
```

Comment:
this comment appears to be very misleading
```suggestion
    /// @dev used for administrative tasks not requiring governance approval and SIP; meant to be dedicated multisig(s)
```

Reply:
Fixed in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/c4c43c22e2dc30a936960e01e9a8acd6e2053616)